docs/development/style-guide.md

```diff
@@ -21,6 +21,33 @@ Second sentence on a new line.
 And so on.
 ```
 
+## Avoid contractions
+
+Avoid:
+
+```markdown
+don't
+won't
+shouldn't
+wouldn't
+manager's
+file's
+Renovate's
```

Comment:
```suggestion
```

I'm happy to keep possessives like `Renovate's package manager` or `Renovate's style guide`.

We should probably avoid possessives like `manager's` and `file's`. With `Renovate's` its clear we refer to the bot, but with `manager's` or `file's` it's not so easy.